.github_changelog_generator

```diff
@@ -18,8 +18,6 @@
 # under the License.
 #
 
-# point to the old changelog in apache/arrow
-front-matter=For older versions, see [apache/arrow/CHANGELOG.md](https://github.com/apache/arrow/blob/master/CHANGELOG.md)\n
```

Comment:
not needed anymore because it's already included in the 5.0.0 changelog and we are appending 6.0.0 changelog on top of it.